packages/intent/src/commands/resolve.ts

```
@@ -0,0 +1,51 @@
+import { fail } from '../cli-error.js'
+import {
+  scanOptionsFromGlobalFlags,
+  type GlobalScanFlags,
```

ESLint

[error] Member 'GlobalScanFlags' of the import declaration should be sorted alphabetically.

(sort-imports)